app/services/job_lister.rb

```diff
@@ -2,10 +2,14 @@
 
 require 'sidekiq/api'
 
-# Lists the FetchMonths for the currently running Fetch jobs.
-class FetchJobLister
-  def self.list
-    new.list
+# Lists the ActiveRecord objects for the currently running jobs.
+class JobLister
```

Comment:
Renamed since this will also retrieve RegistrationJobs (in addition to FetchMonths).